src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -5,10 +5,6 @@
  *
  * (c) Fabien Potencier <fabien@symfony.com>
  *
- * This code is partially based on the Rack-Cache library by Ryan Tomayko,
- * which is released under the MIT license.
- * (based on commit 02d2b48d75bcb63cf1c0c7149c077ad256542801)
```

Comment:
please keep this notice

Reply:
I had it initially, but `fabbot.io` said to apply a patch that removed it

Reply:
yep, fabbot is not bullet proof, that's one false positive.